src/main/java/gyro/aws/cloudfront/CloudFrontResource.java

```diff
@@ -698,4 +729,15 @@ private DistributionConfig distributionConfig() {
 
         return builder.build();
     }
+
+    private void applyMonitoringSubscription(CloudFrontClient client, boolean isUpdate) {
+        if (getMonitoringSubscription()) {
```

Comment:
we should add a null check here since it'll throw a `NullPointerException` if `getMonitoringSubscription()` is not set.
```suggestion
        if (getMonitoringSubscription() != null && getMonitoringSubscription()) {
```